src/NUnitFramework/framework/Constraints/ConstraintExpression.cs

```diff
@@ -8,10 +8,10 @@
 // distribute, sublicense, and/or sell copies of the Software, and to
 // permit persons to whom the Software is furnished to do so, subject to
 // the following conditions:
-// 
+//
```

Comment:
I agree to be honest, it makes reviewing harder.

I'd be in favour of doing a single PR to 'fix the whitespace' - to clear up anything that might get autocorrected in PRs like this. 🙂